From 753cd56228b4102182b47d8231f89e9ae48e0861 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Wed, 12 Oct 2022 19:25:23 -0500
Subject: [PATCH 2/3] Revert "msvcrt: Enable LFH for internal heaps."

This reverts commit bf1b3d09a0163ec29dd737a23d45ce7ab0470a1c.

CW-Bug-Id: #21364
---
 dlls/msvcrt/heap.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/dlls/msvcrt/heap.c b/dlls/msvcrt/heap.c
index 11111111111..11111111111 100644
--- a/dlls/msvcrt/heap.c
+++ b/dlls/msvcrt/heap.c
@@ -530,9 +530,7 @@ int CDECL _set_sbh_threshold(size_t threshold)
 
   if(!sb_heap)
   {
-      ULONG hci = 2;
       sb_heap = HeapCreate(0, 0, 0);
-      HeapSetInformation(sb_heap, HeapCompatibilityInformation, &hci, sizeof(hci));
       if(!sb_heap)
           return 0;
   }
@@ -831,9 +829,7 @@ int CDECL wmemcpy_s(wchar_t *dest, size_t numberOfElements,
 
 BOOL msvcrt_init_heap(void)
 {
-    ULONG hci = 2;
     heap = HeapCreate(0, 0, 0);
-    HeapSetInformation(heap, HeapCompatibilityInformation, &hci, sizeof(hci));
     return heap != NULL;
 }
 
